code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -265,20 +265,28 @@
 	for(var/mob/living/carbon/C in hearers(4))
 		if(C == user)
 			continue
+		if(!affects(C))
+			continue
 		if(ishuman(C))
 			var/mob/living/carbon/human/H = C
 			if(H.check_ear_prot() >= HEARING_PROTECTION_TOTAL)
 				continue
-		if(!affects(C))
-			continue
-		to_chat(C, "<span class='warning'><font size='3'><b>You hear a ear piercing shriek and your senses dull!</font></b></span>")
+			if(H.check_ear_prot() >= HEARING_PROTECTION_MINOR)
+				to_chat(C, "<span class='warning'><font size='3'><b>You hear a dull shriek and instinctively cover your ears!</font></b></span>")
+				C.Confused(10)
+				C.Stuttering(10)
+				C.Jitter(50)
```

Comment:
Nitpick, but use `H` here to be clear the mob is a human.
```suggestion
				to_chat(H, "<span class='warning'><font size='3'><b>You hear a dull shriek and instinctively cover your ears!</font></b></span>")
				H.Confused(10)
				H.Stuttering(10)
				H.Jitter(50)
```

Reply:
To my understanding, C stands for carbon - and H is used for human in the ear check. 

Switching C for H meant I had to remove `var/mob/living/carbon/human/H = C`, and broke the code at line 272.

Reply:
Do not remove the `var/mob/living/carbon/human/H = C` line. That is required.

Reply:
What Steel means here is that, since you've casted `C` to `H` just above, it makes more sense in terms of readability if all of these subsequent procs also use `H` where they can.
![image](https://user-images.githubusercontent.com/57483089/127751883-d906bf5c-e0d0-46bc-9f85-36008d6f144e.png)
vs
![image](https://user-images.githubusercontent.com/57483089/127751894-1be95cae-e9f3-4192-acb8-191105fb29fb.png)
You should be able to just commit the suggested changes to do it.